### PR TITLE
AFGS-565 Added ClaimDTO class used for read/query operations.

### PR DESCRIPTION
**Pull request by dwybourn**

### CODE REVIEW

File: api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimDTO.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Value
+@Builder
+@AllArgsConstructor(onConstructor_ = {@JsonCreator})
+@ApiModel(description = "An existing claim for Apply for healthy start.")
+public class ClaimDTO {
+
+    @ApiModelProperty(notes = "The claim's id as stored in the database", example = "96c3f8c0-f6d9-4ad4-9ed9-72fcbd8d692d")
```
Comment (YetAnotherMatt) [inline api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimDTO.java]: Not sure we should be mentioning the database, how about `The unique id of the claim`

File: api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimDTO.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    @ApiModelProperty(notes = "household identifier returned from HMRC", example = "4c697b0a-5709-423b-8e31-b63796cdca7f")
+    @JsonProperty("hmrcHouseholdIdentifier")
+    private String hmrcHouseholdIdentifier;
+
+    @ApiModelProperty(notes = "card id return from the card provider", example = "8b66a86f-778e-47f9-b998-a042476ef2c0 \n")
+    @JsonProperty("cardAccountId")
+    private String cardAccountId;
+
+    @ApiModelProperty(notes = "the current card status", example = "ACTIVE, PENDING_CANCELLATION, SCHEDULED_FOR_CANCELLATION, CANCELLED")
```
Comment (YetAnotherMatt) [inline api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimDTO.java]: Don't think we need to list all the values - that should be provided by swagger (fingers crossed)

File: api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimDTO.java (hunk 1 above)
Comment (dwybourn) [reply api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimDTO.java]: Updated.

File: api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimDTO.java (hunk 2 above)
Comment (dwybourn) [reply api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimDTO.java]: Removed examples for enum values.

File: api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimDTO.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+    @ApiModelProperty(notes = "Household identifier returned from DWP", example = "dd77e887-49cf-43cd-b974-b17273602018")
+    @JsonProperty("dwpHouseholdIdentifier")
+    private String dwpHouseholdIdentifier;
+
+    @ApiModelProperty(notes = "Household identifier returned from HMRC", example = "4c697b0a-5709-423b-8e31-b63796cdca7f")
+    @JsonProperty("hmrcHouseholdIdentifier")
+    private String hmrcHouseholdIdentifier;
+
+    @ApiModelProperty(notes = "Card id return from the card provider", example = "8b66a86f-778e-47f9-b998-a042476ef2c0 \n")
```
Comment (mikeSmith08) [inline api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimDTO.java]: Minor - `returned`.

Comment (dwybourn) [reply api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimDTO.java]: Will fix in next PR.

**Pull request merged**